test/acl.test.js

```diff
@@ -6,6 +6,9 @@
 /*global describe, beforeEach, afterEach, it */
 'use strict';
 var path = require('path');
+var SG = require('strong-globalize');
+SG.SetRootDir(path.join(__dirname, '..'), {autonomousMsgLoading: 'all'});
```

Comment:
I think you should `require('../lib/globalize')` here too, unless I am missing something?